certora/specs/SocialRecoveryModule.spec

```
@@ -541,7 +543,9 @@ rule recoveryRequestsChange(env e, calldataarg args, method f) {
 // This rule verifies that is the confirmedHashes change, it must be one of the following functions:
 // - confirmRecovery(...)
 // - multiConfirmRecovery(...)
-rule confirmedHashesChange(env e, calldataarg args, method f, bytes32 hash, address guardian) {
+rule confirmedHashesChange(method f, bytes32 hash, address guardian) {
+    env e;
+    calldataarg args;
     bool confirmedHashBefore = currentContract.confirmedHashes[hash][guardian];
 
     f(e, args);
```

Super nitty, but I think it "looks" nicer like this:

```suggestion
    bool confirmedHashBefore = currentContract.confirmedHashes[hash][guardian];

    env e;
    calldataarg args;
    f(e, args);
```

---

Done 👍🏾